openapi/p4pa-classification.openapi.yaml

```
@@ -1117,3 +1170,17 @@ components:
           type: array
           items:
             type: string
+    CreateAssessmentsDetail:
+      type: object
+      required:
+        - assessmentRegistryId
+        - iudSet
+      properties:
+        assessmentRegistryId:
+          type: integer
+          format: int64
+        iudSet:
```

I would not "complicate" the name with the type, let's just use the plural form
```suggestion
        iuds:
```